bot/exts/evergreen/minesweeper.py

```diff
@@ -37,7 +38,7 @@ class CoordinateConverter(commands.Converter):
     async def convert(self, ctx: commands.Context, coordinate: str) -> typing.Tuple[int, int]:
```

Comment:
I think this converter should be moved to `bot/utils/converters.py` (and others too, if I've missed any)

Reply:
Resolved in https://github.com/python-discord/sir-lancebot/pull/718/commits/888f83170bce384315a0a5c47b87b7ffbfb5d8ea.